tests/dmstest.yml

```diff
@@ -121,3 +133,33 @@ DMSDocument:
     EmbargoUntilPublish: true
     Folder: 5
     Sets: =>DMSDocumentSet.ds5
+  extraDoc1:
+    Filename: extradoc1
+    Folder: 5
+    Sets: =>DMSDocumentSet.ds6
+    BelongsToSet: 1
+    CreatedByID: 2
+  extraDoc2:
+    Filename: extradoc2
+    Folder: 5
+    CreatedByID: 2
+    BelongsToSet: 0
+  extraDoc3:
+    Filename: extradoc3
+    Folder: 5
+    BelongsToSet: 0
+    CreatedByID: 2
+  docSaveLinkedDocuments1:
+    Filename: saveLinkedDocument1
+    Folder: 5
+    Sets: =>DMSDocumentSet.dsSaveLinkedDocuments
+  docSaveLinkedDocuments1:
+    Filename: saveLinkedDocument1
+    Folder: 5
+    Sets: =>DMSDocumentSet.dsSaveLinkedDocuments
+  docSaveLinkedDocuments1:
```

Comment:
These three fixture definitions just overwrite themselves - delete the duplicates